tempodb/encoding/common/interfaces.go

```diff
@@ -75,6 +75,10 @@ type CompactionOptions struct {
 	BlockConfig        BlockConfig
 	Combiner           model.ObjectCombiner
 
+	// DropObject can be used to drop a trace from the compaction process. Currently it only receives the ID
+	// of the trace to be compacted. If it returns false, the trace will be dropped.
```

Comment:
It should not be: is it returns true, the trace will be dropped? The "DropObject" name of the method suggests the opposite.

Reply:
i updated the comment to say 
```
If the function returns true, the trace will be dropped.
```
it makes sense to me that `DropObject` returning true would cause an object to be dropped.